python/dataproc_templates/hive/util/README.md

```diff
@@ -0,0 +1,55 @@
+# Hive DDL Extractor
+
+This is a PySpark utility to extract DDLs from Hive metastore. Users can use this utility to accelerate the Hive migration journey.
+
```

Comment:
Create a subsection for required subsequent actions:
Usage of BigQuery SQL Translator to create BigQuery DDLs
Guidance on how to load to BigQuery using the BigQuery DDLs and the auxiliary metadata table. 

Adding the links to each documentation